kv/batch.go

```diff
@@ -27,11 +27,36 @@ import (
 	"golang.org/x/net/context"
 )
 
+// window is a key range passed to the truncate function.
+type window struct {
```

Comment:
This is similar to a `roachpb.Span`, but it contains `RKeys` instead of `Keys`. Maybe call it `rSpan` (and the fields `key` and `endKey`) instead of introducing a new word "window"?